Automattic/jetpack
# Social | Remove sharing buttons info from connections modal

Sharing buttons information inside the connections modal is not relevant to social sharing. Thus, we are removing that information

## Proposed changes:

* Remove the sharing buttons information from connections modal for Facebook, LinkedIn and Tumblr
* Update the modal title which I forgot to update in #37571

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Ensure that feature flag is ON - `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );`
- Test the following for both Jetpack and Social plugin
    - Goto connections management UI
    - Open the modal
    - Confirm that you see the updated title in the modal - `Manage Jetpack Social connections`
    - Delete all the connections if you have any
    - Open the information panel for Facebook, LinkedIn and Tumlr
    - Confirm that you no longer see the information and the image related to sharing buttons

<table>
    <tbody>
        <tr>
            <th>BEFORE</th>
            <th>AFTER</th>
        </tr>
        <tr>
            <td>
<img width="1050" alt="Screenshot 2024-05-28 at 11 44 28 AM" src="https://github.com/Automattic/jetpack/assets/18226415/2313436a-1113-4ae2-b2fc-f36007335e06">
</td>
            <td>
<img width="1050" alt="Screenshot 2024-05-28 at 11 53 03 AM" src="https://github.com/Automattic/jetpack/assets/18226415/7a8b269b-660e-4297-a42f-34e85e13f9a7">

</td>
        </tr>
    </tbody>
</table>
